modules/mw.cx.TargetArticle.js

```
@@ -208,13 +214,46 @@ mw.cx.TargetArticle.prototype.publishSection = function () {
+		console.log("_____");
+		console.log("local result: " + JSON.stringify(publishResult.save_result_all.result));
+		console.log("mdwiki_result: " + JSON.stringify(publishResult.save_result_all.mdwiki_result));
+	} else {
+		console.log(JSON.stringify(publishResult));
+	}
+	// {"result":"error","edit":{"error":"noaccess","username":"Mr. Ibrahem"}}
 	if (publishResult.result === 'success') {
-		this.translation.setTargetURL(publishResult.targeturl);
-		return this.publishComplete(publishResult.targettitle || null);
+		var targeturl = publishResult.targeturl;
+		if (this.sourceLanguage === "mdwiki" && publishResult.published_to != "local") {
```

ESLint

[error] There must be a space after this paren.

(space-in-parens)